worker/src/test/java/gov/cms/ab2d/worker/config/AutoScalingServiceTest.java

```diff
@@ -23,7 +23,6 @@
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
 
-
```

Comment:
This comment applies here and many other places.

No objections to whitespace cleanups per-se.  I just have a preference to do those sorts of things in their own PR so as not to muddle things.  Whitespace/formatting PRs are trivial to approve and keeps the body count down for the substantial reviews.

Reply:
Just a product of doing 3 massive different merges with master.